spec/rakuten_web_service/gora/course_spec.rb

```diff
@@ -0,0 +1,87 @@
+# encoding: utf-8
+
+require 'spec_helper'
+require 'rakuten_web_service'
+
+describe RakutenWebService::Gora::Course do
+  let(:endpoint) { 'https://app.rakuten.co.jp/services/api/Gora/GoraGolfCourseSearch/20131113' }
+  let(:affiliate_id) { 'dummy_affiliate_id' }
+  let(:application_id) { 'dummy_application_id' }
+  let(:expected_query) do
+    {
+        :affiliateId => affiliate_id,
+        :applicationId => application_id,
+        :keyword => '軽井沢'
+    }
+  end
+
+  before do
+    RakutenWebService.configuration do |c|
+      c.affiliate_id = affiliate_id
+      c.application_id = application_id
+    end
+  end
+
+  describe '.search' do
+    before do
+      response = JSON.parse(fixture('gora/course_search_with_Karuizawa.json'))
+      @expected_request = stub_request(:get, endpoint).
+          with(:query => expected_query).to_return(:body => response.to_json)
+
+      response['page'] = 2
+      response['first'] = 31
+      response['last'] = 60
+      @second_request = stub_request(:get, endpoint).
```

Comment:
Ah, got it.
So I fixed the test to expect not to be called `@second_request` and added a test for `#all`